clang/arm has this issue where it generates call to __mulodi4
which is only in compiler-rt and we end with errors

/usr/src/debug/m4/1.4.18-r0/build/src/../../m4-1.4.18/lib/xalloc.h:107: undefined reference to `__mulodi4'


Index: m4-1.4.18/lib/xalloc-oversized.h
===================================================================
--- m4-1.4.18.orig/lib/xalloc-oversized.h
+++ m4-1.4.18/lib/xalloc-oversized.h
@@ -52,7 +52,7 @@ typedef size_t __xalloc_count_type;
 #elif ((5 <= __GNUC__ \
         || (__has_builtin (__builtin_mul_overflow) \
             && __has_builtin (__builtin_constant_p))) \
-       && !__STRICT_ANSI__)
+       && !__STRICT_ANSI__ && !__clang__)
 # define xalloc_oversized(n, s) \
    (__builtin_constant_p (n) && __builtin_constant_p (s) \
     ? __xalloc_oversized (n, s) \
